### ISSUE TITLE
Connections on browser start for favicons of search providers

### ISSUE THREAD

**Issue by bbondy** [brave/browser-laptop#5840]
A user from support claimed that we made connections to the search providers, probably to download the favicons. These should only be downloaded if they are used.

https://linkbubble.zendesk.com/agent/tickets/6753

```
action: deny 
direction: outgoing 
process: /Applications/Brave.app/Contents/MacOS/Brave 
owner: me 
destination: domain wikimedia.org 
port: any 
protocol: any 
help: On 14 Nov 2016, Brave tried to establish a connection to meta.wikimedia.org on TCP port 443 (https). The request was denied automatically during Silent Mode.

action: deny 
direction: outgoing 
process: /Applications/Brave.app/Contents/MacOS/Brave 
owner: me 
destination: domain startpage.com 
port: any 
protocol: any 
help: On 14 Nov 2016, Brave tried to establish a connection to startpage.com on TCP port 443 (https). The request was denied automatically during Silent Mode.

action: deny 
direction: outgoing 
process: /Applications/Brave.app/Contents/MacOS/Brave 
owner: me 
destination: domain mozilla.net 
port: any 
protocol: any 
help: On 14 Nov 2016, Brave tried to establish a connection to developer.cdn.mozilla.net on TCP port 443 (https). The request was denied automatically during Silent Mode.

action: deny 
direction: outgoing 
process: /Applications/Brave.app/Contents/MacOS/Brave 
owner: me 
destination: domain youtube.com 
port: any 
protocol: any 
help: On 14 Nov 2016, Brave tried to establish a connection to www.youtube.com on TCP port 443 (https). The request was denied automatically during Silent Mode.

action: deny 
direction: outgoing 
process: /Applications/Brave.app/Contents/MacOS/Brave 
owner: me 
destination: domain duckduckgo.com 
port: any 
protocol: any 
help: On 14 Nov 2016, Brave tried to establish a connection to duckduckgo.com on TCP port 443 (https). The request was denied automatically during Silent Mode.

action: deny 
direction: outgoing 
process: /Applications/Brave.app/Contents/MacOS/Brave 
owner: me 
destination: domain bing.com 
port: any 
protocol: any 
help: On 14 Nov 2016, Brave tried to establish a connection to www.bing.com on TCP port 443 (https). The request was denied automatically during Silent Mode.

action: deny 
direction: outgoing 
process: /Applications/Brave.app/Contents/MacOS/Brave 
owner: me 
destination: domain fastly.net 
port: any 
protocol: any 
help: On 14 Nov 2016, Brave tried to establish a connection to brave-laptop-updates.global.ssl.fastly.net on TCP port 443 (https). The request was denied automatically during Silent Mode.

action: deny 
direction: outgoing 
process: /Applications/Brave.app/Contents/MacOS/Brave 
owner: me 
destination: domain amazonaws.com 
port: any 
protocol: any 
help: On 14 Nov 2016, Brave tried to establish a connection to s3.amazonaws.com on TCP port 443 (https). The request was denied automatically during Silent Mode.

action: deny 
direction: outgoing 
process: /Applications/Brave.app/Contents/MacOS/Brave 
owner: me 
destination: domain google.com 
port: any 
protocol: any 
help: On 14 Nov 2016, Brave tried to establish a connection to www.google.com on TCP port 443 (https). The request was denied automatically during Silent Mode.

action: deny 
direction: outgoing 
process: /Applications/Brave.app/Contents/MacOS/Brave 
owner: me 
destination: domain bitgo.com 
port: any 
protocol: any 
help: On 14 Nov 2016, Brave tried to establish a connection to www.bitgo.com on TCP port 443 (https). The request was denied automatically during Silent Mode.

action: deny 
direction: outgoing 
process: /Applications/Brave.app/Contents/MacOS/Brave 
owner: me 
destination: domain brave.com 
port: any 
protocol: any 
help: On 14 Nov 2016, Brave tried to establish a connection to laptop-updates.brave.com on TCP port 443 (https). The request was denied automatically during Silent Mode.

action: deny 
direction: outgoing 
process: /Applications/Brave.app/Contents/MacOS/Brave 
owner: me 
destination: domain github.com 
port: any 
protocol: any 
help: On 14 Nov 2016, Brave tried to establish a connection to assets-cdn.github.com on TCP port 443 (https). The request was denied automatically during Silent Mode.

action: deny 
direction: outgoing 
process: /Applications/Brave.app/Contents/MacOS/Brave 
owner: me 
destination: domain amazon.com 
port: any 
protocol: any 
help: On 14 Nov 2016, Brave tried to establish a connection to www.amazon.com on TCP port 443 (https). The request was denied automatically during Silent Mode.

action: deny 
direction: outgoing 
process: /Applications/Brave.app/Contents/MacOS/Brave 
owner: me 
destination: domain sstatic.net 
port: any 
protocol: any 
help: On 14 Nov 2016, Brave tried to establish a connection to cdn.sstatic.net on TCP port 443 (https). The request was denied automatically during Silent Mode.

action: deny 
direction: outgoing 
process: /Applications/Brave.app/Contents/MacOS/Brave 
owner: me 
destination: domain yahoo.com 
port: any 
protocol: any 
help: On 14 Nov 2016, Brave tried to establish a connection to search.yahoo.com on TCP port 443 (https). The request was denied automatically during Silent Mode.

action: deny 
direction: outgoing 
process: /Applications/Brave.app/Contents/MacOS/Brave 
owner: me 
destination: domain twitter.com 
port: any 
protocol: any 
help: On 14 Nov 2016, Brave tried to establish a connection to www.twitter.com on TCP port 443 (https). The request was denied automatically during Silent Mode.
```

**Comment by Liunkae** [comment]
The favicons are downloaded when loading the Search section of Preferences.

**Comment by bsclifton** [comment]
We'll always have the first time outbound connections... but like @Liunkae captured, the caching for these will be fixed if we fix #2697 :smile:
